build/gulp/vendor.js

```diff
@@ -49,23 +50,30 @@ const VENDORS = [
     },
     {
         path: '/turndown/lib/turndown.browser.umd.js'
+    },
+    {
+        path: '/devexpress-diagram/dx.diagram.js'
+    },
+    {
+        path: '/devexpress-diagram/dx.diagram.css'
     }
 ];
 
 gulp.task('vendor', function() {
     return merge.apply(this, VENDORS.map(function(vendor) {
+        const destinationPath = vendor.path.endsWith(".css") ? DESTINATION_CSS_PATH : DESTINATION_JS_PATH;
         let sourceConfig = vendor.base ? { base: PACKAGES_SOURCE + vendor.base } : null;
-        let stream = gulp.src(PACKAGES_SOURCE + vendor.path, sourceConfig).pipe(gulp.dest(DESTINATION_PATH));
+        let stream = gulp.src(PACKAGES_SOURCE + vendor.path, sourceConfig).pipe(gulp.dest(destinationPath));
 
```

Comment:
I think we should stop here for the `css` files.

Reply:
ok

Reply:
done